Cargo.toml

```diff
@@ -9,6 +9,8 @@ members = [
   "oak_client",
   "oak_containers_stage1",
   "oak_core",
+  "oak_containers_example_trusted_application",
```

Comment:
I think instead of `example` we need to put `hello_world`

So for anyone opening the repo it will be easy to find Hello World (this is probably what all people search in a new repository)

Reply:
fair, sgtm, will update